pkgs/servers/lenpaste/default.nix

```
@@ -0,0 +1,30 @@
+{ lib, buildGoModule, fetchFromGitea, git }:
+
+buildGoModule rec {
+  pname = "lenpaste";
+  version = "1.3";
+
+  src = fetchFromGitea {
+    domain = "git.lcomrade.su";
+    owner = "root";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-d+FjfEbInlxUllWIoVLwQRdRWjxBLTpNHYn+oYU3fBc=";
+  };
+
+  buildInputs = [ git ];
+
+  ldflags = "-w -s -X main.Version=${version}";
+
+  fixupPhase = ''
+    mv $out/bin/lenpaste-unwrapped $out/bin/lenpaste
+  '';
+
+  vendorSha256 = "sha256-PL0dysBn1+1BpZWFW/EUFJtqkabt+XN00YkAz8Yf2LQ=";
+
+  meta = with lib; {
+    description = "Open source analogue of pastebin.com. There is its own API.";
+    homepage = "https://git.lcomrade.su/root/lenpaste";
+    license = licenses.agpl3;
```

Seems to be AGPL v3 or later, as the source files contain the necessary "_or (at your option) any later version_" section.

```suggestion
    license = licenses.agpl3Plus;
```